### ISSUE TITLE
Add Recents

### ISSUE THREAD

**Issue by zhanghai** [zhanghai/MaterialFiles#146]

**Comment by arshit09** [comment]
This is the only thing holding me back from completely switching to Material Files

**Comment by mizu-048** [comment]
any updates?

**Comment by HiroDane** [comment]
Will this ever get added?
It is literally the only thing that is stopping me from deleting my default file manager.

**Comment by rayn000** [comment]
> Will this ever get added? It is literally the only thing that is stopping me from deleting my default file manager.

Yes, same here. Please add it ASAP. zhanghai opened this issue on 2019 himself.

**Comment by metamacro** [comment]
This would be a really nice and useful feature. Have there been any advances?
Amazing project nevertheless! Thank you!

**Comment by SudoProdgen** [comment]
Any updates? I don't switch because I miss this feature from fossify files

**Comment by arshit09** [comment]
Thanks for the comment @SudoProdgen. That gave me the spark to add the feature, as now it's possible using Claude, by myself. I've tested the feature thoroughly as well. I hope the maintainer approves my PR #1631. Meanwhile, I've built the APK on my fork; you can download it from here: https://github.com/arshit09/MaterialFiles/releases
